database/index.js

```diff
@@ -4,6 +4,8 @@ mongoose.connect('mongodb://localhost/review', {
   useUnifiedTopology: true
 });
 
+mongoose.set('useFindAndModify', false);

```

Comment:
### ORMs and performance
ORMs make database manipulation a breeze, but they also slow down queries slightly. This difference is usually marginal enough that it doesn't make a huge difference, but since we're attempting to scale it could become an issue down the line.

I've personally made the decision to work with raw Postgres queries. It's been a nightmare so far. If you don't feel comfortable switching over to raw queries, you can probably hold off and see if it becomes an issue for you later. It might not even be a problem, but I want to get as close to that 10k RPS marker as I can so I'm giving it a shot.